config/site_settings.yml

```diff
@@ -1988,11 +1988,9 @@ developer:
   enable_experimental_sidebar_hamburger:
     default: false
     client: true
-    hidden: true
   enable_sidebar:
-    default: false
+    default: true
```

Comment:
Just to be 100% sure. Should this be true by default?

Reply:
Yup :+1: